ffcx/codegeneration/utils.py

```diff
@@ -7,7 +7,7 @@
 
 import typing
 
-import numpy as _np
+import numpy as np
 import numpy.typing as _npt
```

Comment:
Manually change it to `npt`, for consistency with the automatic change `_np` -> `np`?